pkg/metrics/server.go

```diff
@@ -73,7 +73,8 @@ func (s *Server) Run(ctx context.Context) error {
 		operatorv1alpha1.DelegatedAuthorization{},
 		s.kubeConfig,
 		s.kubeClient,
-		nil, // disable leader election
+		nil,   // disable leader election
+		false, // disable http2
```

Comment:
Is this actually part of the PR? I have nothing against this change, just wondering if this slipped in here by accident?

Reply:
Ya, because change in func signature

Reply:
:facepalm: of course.